android/src/main/java/com/amplitude/android/DefaultTrackingOptions.kt

```diff
@@ -24,6 +23,7 @@ constructor(
             )
 
         @JvmField
+        @Deprecated("Use AutocaptureOptions instead", ReplaceWith("AutocaptureOptions(sessions = false)"))
```

Comment:
The same as above. Better to have all 
```suggestion
        @Deprecated("Use AutocaptureOptions instead", ReplaceWith("AutocaptureOptions(appLifecycles = false, deepLinks = false, screenViews = false, sessions = false)"))
```